### ISSUE TITLE
show current window tabs number

### ISSUE THREAD

**Issue by radioterror** [BenjiWiebe/tab-count#1]
The best tab counter if u add (or tell me how to do):
 - show only current window tabs number
 - takes the same space (size of numbers) when there is only one number and when there are two

Thanks

**Comment by BenjiWiebe** [comment]
@radioterror I believe it does take the same space when there is one number vs two. Yes, the font size gets smaller, but that is so it doesn't take more space on your tab bar.

As for the current window count only, I'll consider adding it. I didn't think anyone besides myself would ever use this, but apparently I was wrong. :)

**Comment by BenjiWiebe** [comment]
I don't think it is currently possible. If anyone knows otherwise, let me know or open a pull request, and I'll re-open the issue.

 The issue is that it doesn't seem to be possible to have a per-window "browser action" (toolbar button). Therefore I can't have per-window tab counts.

**Comment by BenjiWiebe** [comment]
I closed this too soon. I think a compromise is possible, and that would be displaying the tab count for the *current* *focused* window. I don't think I can make it show a per-window count though, due to limitations of the API.

**Comment by ssokolow** [comment]
Current focused window would probably be better than nothing for me (I'd have to try it to be 100% sure).

I advise opening a feature request on Bugzilla toget discussion started among them.

**Comment by BenjiWiebe** [comment]
@ssokolow I might do that. You are welcome to help, as I didn't even intend to actually *use* this addon. It just turned out fairly well anyways. :)

EDIT: might do that as in, starting a bugzilla...discussion...

**Comment by ssokolow** [comment]
I don't mind filing the bug, given that i've done it for a lot of other things I've noticed during my preparation to migrate off of Firefox 52 ESR. It's not difficult.

I'm just concerned that it might come across as a little presumptuous for an end-user to file such a feature request on behalf of an extension when he has neither experience with the extension's codebase nor a demonstrated intent to take responsibility for writing the patches necessary to make use of said new feature.

**Comment by ssokolow** [comment]
It turns out that someone else had already filed a bug and it got fixed while I was dawdling around with my disorganized mess of too many TODO tabs.

https://bugzilla.mozilla.org/show_bug.cgi?id=1419893

I found out today because Planet Mozilla syndicates the list of new extension API features added in each Firefox version and Firefox 62 added an optional `windowId` parameter to the `browserAction` methods.

https://blog.mozilla.org/addons/2018/07/05/extensions-in-firefox-62/
